src/components/Auth/Login/Login.react.js

```diff
@@ -145,6 +145,23 @@ class Login extends Component {
             loading: false,
           });
         });
+      actions
```

Comment:
This implementation is buggy. The actions are async are the `showAdminService` API returns false. Make a separate function in this component for checking Admin and make the function call only after successful login.